program/CORE-V IDE prelminary project proposal.md

```diff
@@ -0,0 +1,251 @@
+- Project Plan Available (PPA) is proposed for mid-October 2020
+- **First Integration at the end of 2020** with the aim to bring up a first prototype. The first prototype would enable the creation of a "hello world application."  The CORE-V IDE may be demonstratable at that point.
+- As the project progresses, the intent is to release it on an ongoing basis the Eclipse Foundation cadence, i.e. 4x per year.
+- The **First version to be released** could be 3Q of 2021.
+
+## OpenHW Members/Participants committed to participate in CORE-V IDE project
+1. Ashling. Ashling will contribute IDE expertise and coding resources (integration of embedded CDT with tool chain, debug support).
+2. Alexander Fedorov will contribute overall architecture and technical leadership, release engineering, and collaboration with Eclipse IDE/CDT ecosystem.
+3. Embecosm will contribute CORE-V compiler tool chain and packaging or stabilization if necessary
+4. To be checked: Alibaba, CMC, Thales
+
+## Technical Project Leader(s) (TPLs)
+The following persons will investigate and resolve overall technical issues related to the IDE
+1. Alexander Fedorov  
+2. Hugh O'Keeffe or an alternative person from Ashling  
+
+## Project Manager (PM)
+The following is tentative:
+- Florian Zaruba of OpenHW staff will act as project manager
+- Jeremy Bennett of Embecosm and Duncan Bees of OpenHW staff will act as project management mentors.
+The project manager would be responsible, in conjuction with the TPLs, to create the project plan, collect requirements for features, track status and report status to the OpenHW TWG, and identify and resolve issues related to the successful outcome. 
+
+
+## Project Documents
+The following project documents will be created:
+- Preliminary Project Proposal (this document)
+- Project Proposal, an updated version of this document
+- Project Plan
+- Feature Description (initial version here: https://github.com/openhwgroup/core-v-sw/blob/master/planning/openhw-ide-requirement.md)
+- Build and Delivery Description
+
+
+
+
+## Virtual customer
+The project team would like to expore the concept of virtual customer to help provide suggestions on feature requirements, participate in ongoing backlog grooming, and review and feedback on CORE-V IDE as it is released. It is suggested that CMC (Hugh Politt-Smith) can act in this role as the IDE is used with the FPGA implementation of CV32E40P
+
+
+## Summary of requirements
+
+The present document gives a present snapshot of requirements and Preliminary Project Proposal, the requirements document https://github.com/openhwgroup/core-v-sw/blob/master/planning/openhw-ide-requirement.md will be used as the reference going forward. 
+
+### Introduction
+Use case list would be useful
+
+### Initial project requirements 
+
+1. Support for 64 bit Windows only

```

Comment:
I suggest that given the heavy Linux use in the community, that Linux support should be there up front. In particular GCC on Windows a bit unusual

Reply:
Changed to "Support for both Linux and 64 bit Windows"